### PR TITLE
feat: reesttructuring project

### PR DESCRIPTION
**Pull request by ajorquera**
This PR made changes to:

* All code needs to be under `src/` folder, except the cucumber config file. `cucumber.js`
* `*.feature` files can have the tag `@ignore` to ignore feature files. 
* Functionality to use ENV variables need to work. Example: the page URL needs to come from a env variable
* We use yarn instead of npm

### CODE REVIEW

File: .DS_Store
Comment (ajorquera) [inline .DS_Store:1]: Add this type of file to .gitignore

File: yarn.lock
Comment (ajorquera) [inline yarn.lock:1]: Dont delete this file

File: .DS_Store
Comment (acharjeeauntor) [reply .DS_Store:1]: Ok

File: yarn.lock
Comment (acharjeeauntor) [reply yarn.lock:1]: ok

File: .DS_Store
Comment (acharjeeauntor) [reply .DS_Store:1]: Done

File: yarn.lock
Comment (acharjeeauntor) [reply yarn.lock:1]: Done

**Pull request merged**